retriever/lib/datasets.py

```diff
@@ -70,5 +70,36 @@ def license(dataset):
 
 def dataset_licenses():
     """Return set with all available licenses."""
-    license_values = [str(script.licenses[0]['name']).lower() for script in SCRIPT_LIST()]
+    license_values = [
+        str(script.licenses[0]['name']).lower()
+        for script in SCRIPT_LIST()
+        if script.licenses
+    ]
     return set(license_values)
+
+
+def dataset_verbose_list(script_names: list):
```

Comment:
Where are we using this function?

Reply:
It has the same implementation as ```retriever ls -v``` command as you requested.

Reply:
I don't see the function being called

Reply:
It is like the ```datasets(keywords=None, licenses=None)``` function, like the ```datasets``` function returns datasets with specified keywords and licenses, the ```dataset_verbose_list``` prints the verbose of the specified dataset names

Reply:
Any suggestions or changes @henrykironde?

Reply:
You need to use that new function in the main file

Reply:
Got it. Thanks @henrykironde